src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -145,6 +146,19 @@ private static boolean validate(Request req, FeedSource feedSource) {
             );
             return false;
         }
+
+        // If the retrieval method is automatic make sure one of the permitted fetch frequencies has been provided.
```

Comment:
This retrieval methods set pertains to feed transformations, not the `FeedSource#retrievalMethod` field.